runtime/runtime/src/lib.rs

```diff
@@ -2175,7 +2176,8 @@ impl Runtime {
                 &receipt_sink.outgoing_receipts(),
                 &processing_state.stats,
             ) {
-                panic!(
+                tracing::error!(
```

Comment:
Lets revert the PR entirely rather than undoing half of it -- its less risky that way among other things.